unity-client/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarRenderer.cs

```diff
@@ -499,13 +499,23 @@ public void SetVisibility(bool newVisibility)
                 gameObject.SetActive(newVisibility);
         }
 
-        private void HideAll()
+        public void HideAllRenderers()
         {
             Renderer[] renderers = gameObject.GetComponentsInChildren<Renderer>();
 
-            for (int i = 0; i < renderers.Length; i++)
+            foreach (Renderer renderer in renderers)
```

Comment:
We generally avoid using `foreach` when working with arrays because its a bit slower and more GC trash is created. In this case we can live with this because I don't think this is a performance critical code, but take this into account for the future.

Reply:
Reverted the change 😄